release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-adm-upgrade-status-tp"]
+==== Introducing the `oc adm upgrade status` command (Technology Preview)
+
+Previously, users could run the `oc adm upgrade` command to retrieve limited information about the cluster update status. With this release, there is now a tech preview `oc adm upgrade status` command in addition to `oc adm upgrade`, which decouples the status information from `oc adm upgrade` and complements the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Comment:
Reviving [this thread][1] about "tech preview" vs. "Technology Preview" in the "With this release, ..." sentence.  I'm personally agnostic; but wanted the docs folks to at least have consensus among yourselves :)

[1]: https://github.com/openshift/openshift-docs/pull/75679#discussion_r1594563126

Reply:
Sorry! Thought I submitted the fix :) good catch - done!